snuba/datasets/entity_subscriptions/entity_subscription.py

```diff
@@ -88,6 +92,9 @@ def get_entity_subscription_conditions_for_snql(
     def to_dict(self) -> Mapping[str, Any]:
         return {"organization": self.organization}
 
+    def get_partitioning_key(self) -> int:
```

Comment:
Let's implement this method for `GenericMetricsSetsSubscription` and `GenericMetricsDistributionsSubscription` only for now since they are the only entities which need slicing.

Reply:
Ok, this makes sense